File: src/me/xerces/eventbus/annotation/EventHandler.java
package me.xerces.eventbus.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Created by Crysk on 12/01/2015.
 */
@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.METHOD)
public @interface EventHandler {

    EventPriority priority() default EventPriority.LOW;
    
}

File: src/main/java/dev/throwouterror/eventbus/annotation/EventHandler.kt
package dev.throwouterror.eventbus.annotation

@Retention(AnnotationRetention.RUNTIME)
@Target(AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER)
annotation class EventHandler(val priority: EventPriority = EventPriority.LOW)